src/Collector/MethodCallCollector.php

```diff
@@ -109,28 +109,11 @@ public function processNode(Node $node, Scope $scope): ?array
         }
 
         $declaringClassName = $reflectionMethod->getDeclaringClass()->getName();
-
-        // Do not find template variables in external classes
-        if ($this->isExternalCall($declaringClassName, $calledMethodName)) {
```

Comment:
Why is this removed?

Reply:
I think it is unreachable because of collector paths added in previous commits